Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -87,9 +95,32 @@ namespace WhiteBox
             enable = true;
             break;
         }
+        
+        switch (m_transformType)
+        {
+        case WhiteBox::TransformType::Translation:
+            SetViewportUiClusterActiveButton(m_transformClusterId, m_transformTranslateButtonId);
+            break;
+        case WhiteBox::TransformType::Rotation:
+            SetViewportUiClusterActiveButton(m_transformClusterId, m_transformRotateButtonId);
+            break;
+        case WhiteBox::TransformType::Scale:
+            SetViewportUiClusterActiveButton(m_transformClusterId, m_transformScaleButtonId);
+            break;
+        default:
+            break;
+        }
```

Comment:
Maybe just make the cluster on demand. I could place it on the right side of the screen. If it's on the left it can be placed after the mode selection. I can just move all of this into the mode itself

Reply:
Could be a question for sig/ui-ux - it should be possible to stack the cluster to the right of the existing one too - that would be great thanks!